### ISSUE TITLE
How to use the Token

### ISSUE THREAD

**Issue by t-loree** [Mario-35/Stean#6]
In the documentation, there is no example of how to use the token.
![image](https://github.com/user-attachments/assets/79072f29-14d7-474d-99c0-9d6027220b1f)

**Comment by Mario-35** [comment]
Yes it's true and the exemple is bad when admin will be finished I do toso examples.
